FireboltDotNetSdk.Tests/Unit/FireboltCommandTest.cs

```diff
@@ -188,5 +189,64 @@ public void SetParamListDatesTest()
             var expect = commandText.ToString("yyyy-MM-dd HH:mm:ss");
             Assert.That("'" + expect + "'", Is.EqualTo(result));
         }
+
+        [Test]
+        public void TimestampTzTest()
```

Comment:
nit: Consider using nunit testcase https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcase.html

Reply:
Also, please add tests for timezone values with seconds `+11:11:11`, since we can get those as well.
For example, with `time_zone=Asia/Calcutta` execute `1111-01-05 17:04:42.123456`. The response will be `1111-01-05 17:04:42.123456+05:53:28`, which is actually correct for some reason

Reply:
I had to change how the ts was handled but it seems ok now - I added the following test
https://github.com/firebolt-db/firebolt-net-sdk/blob/bf0c8797bf9ce0cc1b53765365e2cf987bd6cb14/FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs#L144-L160